src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -652,16 +663,12 @@ node_types=
 # gateway = 192.168.122.1
 
 [PowerFlex Settings]
-
-# Settings for PowerFlex OS Storage Backend
-enable_powerflex_backend=false
-
 # Method to retrieve RPMs before Installation begins
 # Can be either local or external
 powerflex_rpms_method=local
 
 # Used to override default RPMs path.
-# PowerFlexRpmsPath='/opt/dellemc/powerflex/rpms'
+powerflex_rpms_path='/opt/dellemc/powerflex/rpms'
```

Comment:
Could be conbined with powerflex_gateway_rpm above ?

Reply:
Don't get your point. Please clarify.